plugin/src/main/java/hudson/plugins/swarm/PluginImpl.java

```diff
@@ -70,6 +92,12 @@ public void doCreateSlave(StaplerRequest req, StaplerResponse rsp, @QueryParamet
                 }
                 jenkins.addNode(slave);
             }
+            rsp.setContentType("text/plain; UTF-8");
+            byte[] response = name.getBytes("UTF-8");
+            rsp.setContentLength(response.length);
```

Comment:
may I recommend adding a terminating \r\n (or \n) or perhaps even a `swarmName=blah` rather than just `blah` so that it is easier to evolve this in the future if need be without breaking older clients.